src/SamplesApp/Benchmarks.Shared/Suite/Windows_UI_Xaml_Controls/DependencyPropertyBench/SimpleDPBenchmark.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+
+namespace SamplesApp.Benchmarks.Suite.Windows_UI_Xaml_Controls.GridBench
+{
+	public class SimpleDPBenchmark
+	{
+		private Grid SUT;
+
+		[GlobalSetup]
+		public void Setup()
+		{
+			SUT = new Grid();
+		}
+
+		[Benchmark()]
+		public void DP_Write()
+		{
+			for (int i = 0; i < 100; i++)
```

Comment:
Does unrolling give you nicer numbers here ?

Reply:
Also is a 100 times enough ?

Reply:
Probably. I'm not sure if it's the best way to test this tbh. I'll adjust the test with the next PR to validate more.